### DISCUSSION TITLE
Arrows, fields and unit boundaries

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#220] [General]
Hi Kjell,
I showed your project to a military friend of mine, and he said it's very interesting, but immediately asked me if it's possible to draw map enhancers like lines and fields. I thought it was possible, but i did not find the possibility.
I don't know if I explain myself. I have seen I can add units and subunits till single tank and soldier, but there's no way to add to the map things like minefields, bunkers and fortified lines, artillery bombing area, arrows for advances and movement, unit boundaries at different heyrarchic level, and so on.
Do you plan to do it? Or maybe I simply did not find and it's already possible...
Best Regards again
Federico

**Comment by federicopeyrani1969** [comment]
Ah, and he also asks if you plan to enable printing the scenarios on paper.
Federico

**Reply by kjellmf** [reply]
There is already an issue for this #32 (PDF export)

**Comment by kjellmf** [comment]
You can draw simple point, lines and polygons, but not any control measures like tasks, arrows, minefields etc. I may add support for a few common control measures in the feature, but it is quite complicated to get it right. An excellent tool for drawing control measures is https://www.map.army/
